docs/ref/validators.txt

```diff
@@ -161,14 +161,22 @@ to, or in lieu of custom ``field.clean()`` methods.
 
     .. attribute:: schemes
 
+        .. versionchanged:: 1.10
```

Comment:
This should appear at the bottom of the section with a description of what changed.